nidn/fdtd_integration/compute_spectrum_fdtd.py

```diff
@@ -153,3 +170,28 @@ def _average_along_detector(signal):
             s[2] += p[2] / len(signal[i])
         avg[i] = s
     return avg
+
+
+def _summed_thickness_times_permittivity(thicknesses, permittivity):
+    summed_permittivity = 0
+    for i in range(len(thicknesses)):
+        summed_permittivity += thicknesses[i] * permittivity[:, :, i].real
+    return summed_permittivity
+
+
+def _check_sufficient_timesteps(
+    cfg: DotMap, thicknesses, wavelength, permittivity, scaling
+):
+    recommended_timesteps = int(((
+        cfg.FDTD_free_space_distance
+        + 3 * _summed_thickness_times_permittivity(thicknesses, permittivity)
+        + 2 * wavelength * 1e6
```

Comment:
```suggestion
        + 2 * wavelength * UNIT_MAGNITUDE
```
I think?

Will need
`from ..utils.global_constants import UNIT_MAGNITUDE` in the top

Reply:
Yeah, this will make it consistent with cfg.FDTD_free_space_disatnce and such:)